docs/content/en/docs/clustermgmt/cluster-rebootnode.md

```diff
@@ -17,25 +17,51 @@ Rebooting a cluster node as described here is good for all nodes, but is critica
 If it does go down while running the `boots` service, the Bottlerocket node will not be able to boot again until the `boots` service is restored on another machine. This is because Bottlerocket must get its address from a DHCP service.
 {{% /alert %}}
 
-1. Cordon the node so no further workloads are scheduled to run on it:
+
+1. [Backup cluster]({{< relref "/docs/clustermgmt/cluster-backup-restore/backup-cluster" >}}) 
+
+  This ensures that there is an up-to-date cluster state available for restoration in the case that the cluster experiences issues or becomes unrecoverable during reboot.
+
+2. Verify DHCP lease time will be longer than the maintenance time, and that IPs will be the same before and after maintenance. 
+
+  This step is critical in ensuring the cluster will be viable after cluster reboot. **If this cannot be verified, do not proceed any further**
+
+3. Pause reconciliation of the cluster being shut down. This ensures that the cluster will not try to remediate the nodes when shutting down.
+
+  - add the paused annotation to the EKSA clusters being shut down: 
+  ```bash
+  kubectl --kubeconfig=MANAGEMENT_KUBECONFIG annotate clusters.anywhere.eks.amazonaws.com CLUSTER_NAME anywhere.eks.amazonaws.com/paused=true
+
+  kubectl --kubeconfig=MANAGEMENT_KUBECONFIG -n eksa-system annotate clusters.cluster.x-k8s.io CLUSTER_NAME cluster.x-k8s.io/paused=true
+  ```
+
+4. Perform the following steps order of Worker, Control plane, etcd.
+
+  1. Cordon the node so no further workloads are scheduled to run on it:
 
     ```bash
     kubectl cordon <node-name>
     ```
 
-1. Drain the node of all current workloads:
+  2. Drain the node of all current workloads:
 
    ```bash
    kubectl drain <node-name>
    ```
 
-1. Shut down. Using the appropriate method for your provider, shut down the node.
+  3. Shut down. Using the appropriate method for your provider, shut down the node.
```

Comment:
what does this sentense mean? is it something like
"Shut down the corresponding infrastructure machine of the node using the appropriate method for your provider"
like what does this step mean for vsphere provider?